lang/py/avro/schema.py

```diff
@@ -570,7 +570,8 @@ def __init__(
                     raise avro.errors.InvalidName("An enum symbol must be a valid schema name.")
 
         if len(set(symbols)) < len(symbols):
-            raise avro.errors.AvroException(f"Duplicate symbol: {symbols}")
+            duplicate_symbols = {symbol for symbol in symbols if symbols.count(symbol) > 1}
+            raise avro.errors.AvroException(f"Duplicate enum symbol: {list(duplicate_symbols)[0]}")
```

Comment:
This doesn't match with the issue:
> Improvement removes this defect and shows a message "Duplicate symbol" with the symbol, if it is one, or "Duplicates symbols" with the list of duplicate symbols, if there are more than one symbol.

May I suggest to raise this instead?
```python
raise avro.errors.AvroException(f"Duplicate enum symbol(s): {list(duplicate_symbols)}")
```

Reply:
Hello, @opwvhk!
I used Avro Java format to the output.
See previous PR: https://github.com/apache/avro/pull/1759.